sql/hive/src/main/scala/org/apache/spark/sql/hive/execution/CreateViewAsSelect.scala

```diff
@@ -72,27 +85,40 @@ private[hive] case class CreateViewAsSelect(
         HiveColumn(col.name, HiveMetastoreTypes.toMetastoreType(attr.dataType), col.comment)
       }
     }
+  }
 
-    val columnNames = childSchema.map(f => verbose(f.name))
-
+  private def expandViewText: String = {
```

Comment:
I feel this name (`expandViewText`) is not really suitable. This is the version that does not canonicalize the view definition, right?